quickstart/101-vm-cluster-windows/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "resource_group_name" {
```

Comment:
@TomArcherMsft Shall we include output like host group id and machine scale set id as well?

Reply:
@Jingwei-MS 

How do I access those two values to output them and what can the customer do with those values? 

To keep the sample simple, we limit the output values to those that are dynamically created and/or values that must be used to complete the tutorial. 

For example:
- If we generate an Azure resource name using something like `random_pet`, we output that value so the customer knows what was created. 
- We also output any values that are required parameters to the Azure CLI and Azure PowerShell commands in the "Verify the results" section of the article.